modules/log4j2/pom.xml

```diff
@@ -66,6 +66,12 @@
             <artifactId>log4j-core</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-jcl</artifactId>
```

Comment:
Why do we add a bridge for JUL in the SLF4J, but not here?

Reply:
Done